### DISCUSSION TITLE
Difficult to find the submit review form from the pull request screen

### DISCUSSION THREAD

**Discussion by sharplet** [community/community#4467] [Mobile] [answered]
I consistently find it difficult to locate the right place to submit a review when I'm on the main pull request screen. I think the reason is that while there's a big green merge button, to find the review button you have to navigate to the changed files screen through this row:

![image](https://user-images.githubusercontent.com/147686/124285696-2ae2b580-db1c-11eb-99a4-b6ec7bfb5119.jpeg)

And when you haven't submitted a review yet, the word "review" doesn't even appear in the copy.

Basically, the app seems to really want me to submit my review from the files changed view, but even after looking at the diffs I almost always want to go back and review the conversation first before submitting a review. And if I've previously submitted a review and want to quickly upgrade it to an approval, I want a quick way to do that without having to go back to the diffs.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @sharplet – right now we do have these blue `Review` calls to action, depending on your relationship with the pull request:

![IMG_502669578753-1](https://user-images.githubusercontent.com/1923260/126219051-3469f3fc-b103-42a5-911e-6f33fbc687e7.jpeg)

![IMG_838B5B3B47DE-1](https://user-images.githubusercontent.com/1923260/126219188-1cf361bc-3a36-4bb3-9db3-77b86965832a.jpeg)

![IMG_A77C4A628096-1](https://user-images.githubusercontent.com/1923260/126219264-95f5f54e-56ad-4d75-9718-235bba3da1d8.jpeg)

It's possible for this blue text to not appear if you haven't been requested to review a PR, or aren't a contributor in a repository. Could this be the case for you?

In any case, this behavior does mimic the behavior of GitHub.com in many ways, which requires you to go to the files changed view before leaving a review. We'd have to think longer on if shortcutting that makes sense or if it could lead to potentially bad code review behaviors.

**Reply by sharplet** [reply]
FWIW, the other behaviour on github.com that supports this workflow is that any comments made from the files changed view are collected into a yellow "pending" review on the comments tab. This is where I usually submit the review from, in part because it more closely represents what the author of the PR will see when my review is submitted.

I'll keep an eye out for the blue CTAs. I'm definitely a contributor on the repo I'm thinking of.

**Reply by brianlovin** [reply]
Great point. We should make the "pending" handoff much more clear, thanks for the detail!
